### PR TITLE
[PREVIEW]SSCS-3802 Add whitelist regex to the whatYouDisagreeWith field to all…

### PR DESCRIPTION
**Pull request by iguannago**
Apply to the reason for appealing the same fix that we applied to other reasons for appealing to allow special chars.

### CODE REVIEW

File: test/e2e/page/reasons-for-appealing/reasonsForAppealingOnePageForm.test.js [hunk 1]
```diff
@@ -87,7 +105,9 @@ I see an error in the error summary`, async I => {
   I.see(content.listError);
 });
 
-Scenario(`When I add a reasons then click the add another reason button and enter the least amount 
+// this test is not passing.
+// Ticket was raised to fix it https://tools.hmcts.net/jira/browse/SSCS-3929
+xScenario(`When I add a reasons then click the add another reason button and enter the least amount
```
Comment (davet1985) [inline test/e2e/page/reasons-for-appealing/reasonsForAppealingOnePageForm.test.js]: If this test is failing, does it not mean that the functionality of adding multiple reasons is broken?

File: steps/reasons-for-appealing/reason-for-appealing/content.en.json [hunk 2]
```diff
@@ -28,6 +28,9 @@
         "required": "You need to give at least one reason for appealing",
         "notEnough": "You have not said enough about why you want to appeal. Briefly explain why you disagree with DWP’s decision."
       }
+    },
+    "error": {
+      "invalid": "You've entered a special character. Delete it before continuing"
```
Comment (davet1985) [inline steps/reasons-for-appealing/reason-for-appealing/content.en.json]: Did @Mickyelvino approved this content?

File: test/e2e/page/reasons-for-appealing/reasonsForAppealingOnePageForm.test.js (hunk 1 above)
Comment (iguannago) [reply test/e2e/page/reasons-for-appealing/reasonsForAppealingOnePageForm.test.js]: Yes, that's my understanding as well. I created this ticket for BA and business to prioritise https://tools.hmcts.net/jira/browse/SSCS-3929

File: steps/reasons-for-appealing/reason-for-appealing/content.en.json (hunk 2 above)
Comment (iguannago) [reply steps/reasons-for-appealing/reason-for-appealing/content.en.json]: I am using the very same content that is in Production right now. Therefore, that was agreed with @Mickyelvino  already.

**Pull request merged**